mkdocs.yml

```diff
@@ -7,6 +7,7 @@ nav:
     - Database schema: schema.md
     - Schema loader: schema-loader.md
 - Design: design.md
+- Jepsen: https://github.com/scalar-labs/scalar-jepsen
```

Comment:
Does it work ?

Reply:
I don't know how to check without it being merged. In any case, according to https://www.mkdocs.org/user-guide/configuration/#nav it certainly should.

Reply:
It should work but can you check it with `mkdocs serve` quickly ?
mkdocs can be installed through brew.

Reply:
Works fine with mkdocs, version 1.0.4. It did not, however, with some older version of mkdocs.

Reply:
Hmm, ok. not sure about what version of mkdocs readthedocs uses. We'll see...